### ISSUE TITLE
Infrastructure resources monitoring

### ISSUE THREAD

**Issue by gawertm** [giantswarm/roadmap#3601]
It would be helpful to have an operator to expose VCD/vSphere infrastructure resource usage to get an alert when running low.

(Discussed during retro)

**Comment by gawertm** [comment]
will focus on vSphere with this

**Comment by glitchcrab** [comment]
https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.exporter.vsphere/

**Comment by gawertm** [comment]
@glitchcrab what do you mean with the link above? looks like the exporter is deprecated?

**Comment by glitchcrab** [comment]
> [@glitchcrab](https://github.com/glitchcrab) what do you mean with the link above? looks like the exporter is deprecated?

yes sorry i was just dropping the link here so i didn't forget about it. I talked with Quentin and we came up with a plan as to how to do the monitoring, I just haven't had the time to sit down and get into it yet

**Comment by glitchcrab** [comment]
necessary work is done, just waiting on approvals from Atlas after reworking the PR a bit.
